### PR TITLE
Fix link to download select

### PR DESCRIPTION
**Pull request by northerner**
Using the "text" property as a placeholder for the download select was
breaking the selection of the first item in the list. This is a labeled
select box so a placeholder value was not necessary anyway.

#223

### CODE REVIEW

**Comment by mrhead** [comment]
@northerner What do we show instead of "Choose download"? TinyMCE API is hard to follow. 😳

**Comment by northerner** [comment]
> @northerner What do we show instead of "Choose download"? TinyMCE API is hard to follow. 😳

It just defaults to the first download available, [like podcasts dropdown](https://github.com/memberful/memberful-wp/blob/6397e2ba866ce212c861e1141e9d0e60ba9412b3/wordpress/wp-content/plugins/memberful-wp/js/memberful-wp-tinymce-plugin.js#L69).

**Review by mrhead — APPROVED** [review]
👍
